src/python/foglamp/core/api/configuration.py

```diff
@@ -78,10 +81,14 @@
     """
     category_name = request.match_info.get('category_name', None)
     config_item = request.match_info.get('config_item', None)
+
+    if not category_name or not config_item:
+        raise web.HTTPBadRequest(reason="Both Category Name and Config items are required")
```

Comment:
As per reason msg condition should check with `AND`? otherwise update reason msg accordingly.